client-js/app/scripts/manager.ts

```diff
@@ -53,7 +61,7 @@ export interface ManagerApi {
   serverDisconnected: Observable<ServerDisconnectMessage>;
 }
 
-export class ManagerServiceJsonRpc extends rpc.RpcService implements ManagerApi {
+export class ManagerApiImpl extends rpc.RpcService implements ManagerApi {
```

Comment:
Seems weird to me that the interface gets a nice name, and then the actual class has a "Impl" suffix.